connection/connection.go

```diff
@@ -169,6 +169,17 @@ func (connectionSettings *ConnectionSettings) GetHeaders(state *session.State, e
 
 	state.HeaderJar.SetHeader(host, header)
 
+	restHost, err := connectionSettings.GetRestUrl()
```

Comment:
can't remember, what's the difference between above `host` and the `GetRestUrl`, with and without protocol?

Reply:
Exactly. Tested first with host, resulted in errors so protocol is definitely required here.